modules/docs/mdx/13.0-UPGRADE-GUIDE.mdx

```diff
@@ -126,6 +128,26 @@ The following components are affected:
 </ExternalHyperlink>
 ```
 
+## Pill
+
+**PRs:** [#3104](https://github.com/Workday/canvas-kit/pull/3104)
+
+A few changes have been made to `Pill` to ensure proper accessibility and styles.
+
+- The border color on hover has been updated from `licorice400` to `licorice500` to match our design specs.
```

Comment:
Should we call out the CSS property itself to match the tokens? 

```suggestion
- The `border-color` on hover has been updated from `licorice400` to `licorice500` to match our design specs.
```

Reply:
ill leave this out since I'm talking about actual css property